tests/bsim/bluetooth/ll/bis/prj.conf

```diff
@@ -16,7 +16,7 @@ CONFIG_BT_CTLR_ADV_ISO=y
 CONFIG_BT_CTLR_SYNC_ISO=y
 
 CONFIG_BT_CTLR_ISO_TX_BUFFERS=2
-CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=251
+CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=255
```

Comment:
This change is not required when #63326 is merged upstream. The test case due to change in host in this current PR is making the host to fragment BIS ISO SDU into HCI ISO Data packets, #63326 fixes the Zephyr Controller to correctly generate back the number of completed packets.

Reply:
thank you for the fix
I reverted changes in prj.conf